apps/docs/content/components/Testimonial.mdx

```diff
@@ -57,7 +57,7 @@ A logo (or similar) visual can be provided as an alternative to avatars.
 
```

Comment:
non-blocking: The link to the storybook seems to be broken, can we change it to?

```html
/brand/storybook/?path=/story/components-Testimonial--playground
```